module/VuFind/src/VuFind/ILS/Driver/Demo.php

```diff
@@ -729,6 +730,19 @@ protected function getSimulatedStatus($id, array $patron = null)
 
         if ($json = $this->config['StaticHoldings'][$id] ?? null) {
             foreach (json_decode($json, true) as $i => $status) {
+                if ($status['use_status_class'] ?? false) {
+                    $availability = $status['availability'] ?? false;
+                    if ($status['use_unknown_message'] ?? false) {
+                        $availability = \VuFind\ILS\Logic\AvailabilityStatusInterface::STATUS_UNKNOWN;
```

Comment:
You don't need to use the FQCN here since AvailabilityStatusInterface has a use statement above.

Reply:
fixed